### PR TITLE
folium

### PR DESCRIPTION
**Pull request by ocefpaf**
Folium and all its optional dependencies.

### CODE REVIEW

**Comment by jakirkham** [comment]
Similar ASCII problem I was seeing before. We need to fix the docker image to update `locales` and add some UTF8 encoding (at least the English one). Though we could add as many as we want.

**Comment by ocefpaf** [comment]
I have a `ENV LANG en_US.UTF-8` var in my IOOS Dockerimage, but here we are doing [this](https://github.com/conda-forge/staged-recipes/blob/master/scripts/run_docker_build.sh#L43).

**Comment by jakirkham** [comment]
This is on Debian, but this is what we do to [solve this problem]( https://github.com/jupyter/docker-stacks/blob/master/minimal-notebook/Dockerfile#L37-L38 ) with the [docker-stacks]( https://github.com/jupyter/docker-stacks ). I'll try to write up a workaround proposal.

**Comment by ocefpaf** [comment]
I guess that my question is: do we have a reason to `unset LANG`?

**Comment by jakirkham** [comment]
As your tweak is working, I am borrowing it, as well.

File: recipes/folium/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+    home: https://github.com/python-visualization/folium
+    license: MIT License
+    summary: Make beautiful maps with Leaflet.js and Python
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - BibMartin
```
Comment (ocefpaf) [inline recipes/folium/meta.yaml]: Ping @BibMartin

By adding your name here you'll have commit rights in the conda-forge feedstock for folium. Are you OK with that?

Comment (BibMartin) [inline recipes/folium/meta.yaml]: OK, nom problem.

**Pull request merged**